modules/exampleauth/templates/userclick.twig

```diff
@@ -0,0 +1,63 @@
+{% set pagetitle = 'Continue as persona' %}
+
+{% extends "@core/base.twig" %}
+
+{% block preload %}
+    <link rel="stylesheet" href="{{ asset('css/default.css', 'exampleauth') }}">
+{% endblock %}
+
+{% block content %}
+    {%- if not isProduction %}
```

Comment:
This seems a bit misplaced. This auth source is never for production?

Reply:
<img width="1185" height="192" alt="afbeelding" src="https://github.com/user-attachments/assets/5c2779db-7026-4c49-9bbf-929afb84cf88" />

It will display this warning if `'production' => false` in config.php

Nothing more.. I think Thijs overlooked something, but I this this can be marked as resolved.